topics/climate/tutorials/fates-jupyterlab/tutorial.md

```diff
@@ -291,7 +290,7 @@ The 4 main arguments of create_newcase are explained on the figure below: ![crea
 > ./case.setup
 > ./case.build
 > ./xmlchange STOP_OPTION=nmonths            # set the simulation periods to "years"
-> ./xmlchange STOP_N=1                       # set the length of simulation, i.e, how many years
+> ./xmlchange STOP_N=6                       # set the length of simulation, i.e, how many years
```

Comment:
you've written 1 month above, might be worth it to change too. Additionally the comment here says "for how many years" so not clear if it's months (above) or years here. Same with the STOP_OPTION.

Reply:
Well spotted! It needs to be updated too (6 months everywhere). Thank you @hexylena